pkgs/development/libraries/libtelnet/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, pkgconfig, autoreconfHook, zlib }:
+
+stdenv.mkDerivation rec {
+  name = "libtelnet-${version}";
+  version = "0.21";
+
+  src = fetchFromGitHub {
+    owner = "seanmiddleditch";
+    repo = "libtelnet";
+    rev = "45f2d5cfcf383312280e61c85b107285fed260cf";
+    sha256 = "1lp6gdbndsp2w8mhy88c2jknxj2klvnggvq04ln7qjg8407ifpda";
+  };
+
+  nativeBuildInputs = [ pkgconfig autoreconfHook ];
+  buildInputs = [ zlib ];
+
+  meta = {
+    description = "Simple RFC-complient TELNET implementation as a C library.";
+
+    longDescription =
+      '' Simple RFC-complient TELNET implementation as a C library.
+         Found at https://github.com/seanmiddleditch/libtelnet
```

Comment:
We have a `meta.homepage` field for that. :) As there is no new information in `longDescription`, you can just drop it.